javax-money/src/test/java/com/fasterxml/jackson/datatype/money/CurrencyUnitSchemaSerializerTest.java

```diff
@@ -0,0 +1,25 @@
+package com.fasterxml.jackson.datatype.money;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.module.jsonSchema.JsonSchema;
+import com.fasterxml.jackson.module.jsonSchema.JsonSchemaGenerator;
+import org.junit.Test;
+
+import javax.money.CurrencyUnit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public final class CurrencyUnitSchemaSerializerTest {
```

Comment:
We should remove this test: should NOT include cross-module dependency for tests because this now introduces a build/release dependency between modules.